templates/partial/search_result.html

```diff
@@ -38,43 +34,23 @@ <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
               {{result.metadata.maintainer_display_name}}
             </dd>
           </div>
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">First created:</dt>
-            <dd class="govuk-summary-list__value">
-              {% if result.metadata.creation_date %}
-                {{result.metadata.creation_date | date:"jS F Y"}} ({{result.metadata.creation_date|naturaltime}})
-              {% endif %}
-            </dd>
-          </div>
           <div class="govuk-summary-list__row">
             <dt class="govuk-summary-list__key">Refresh period:</dt>
             <dd class="govuk-summary-list__value">
               TBC
             </dd>
           </div>
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">Retention period:</dt>
-            <dd class="govuk-summary-list__value">
-              {{result.metadata.retention_period_in_days}}
-            </dd>
-          </div>
           <div class="govuk-summary-list__row">
             <dt class="govuk-summary-list__key">Domain name</dt>
             <dd class="govuk-summary-list__value">
               {{result.metadata.domain_name}}
             </dd>
           </div>
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">Tags</dt>
-            <dd class="govuk-summary-list__value">
-              {{ result.tags |join:", " }}
-            </dd>
-          </div>
           {% if result.matches %}
           <div class="govuk-summary-list__row">
             <dt class="govuk-summary-list__key">Matched fields</dt>
             <dd class="govuk-summary-list__value">
-              {{ result.matches|lookup:readable_match_reasons|join:", " }}
+              {{ result.matches|join:", " }}
```

Comment:
```suggestion
              {{ result.matches|lookup:readable_match_reasons|join:", " }}
```
The match reasons dict is missing some items, this pulls through a blank sometimes. Easy to add to.